packages/@react-types/progress/src/index.d.ts

```diff
@@ -63,7 +63,12 @@ export interface ProgressCircleProps extends ProgressBaseProps {
   isIndeterminate?: boolean
 }
 
-export interface AriaProgressCircleProps extends ProgressCircleProps, DOMProps, AriaLabelingProps {}
+export interface AriaProgressCircleProps extends ProgressCircleProps, DOMProps, AriaLabelingProps {
+  /**
+   * Whether the ProgressCircle should be hidden from assistive technology.
+   */
+  'aria-hidden'?: boolean
```

Comment:
I am hoping we don't have to add `aria-hidden` here. 
Maybe we can add a label to the progress that says loading? Or if we cannot do that, I'd rather not have this prop on Progress, maybe we wrap it in a span and add an `aria-hidden` to that instead?
@majornista thoughts?